Formula/a/asdf.rb

```diff
@@ -26,6 +26,7 @@ def install
     libexec.install Dir["*"]
     touch libexec/"asdf_updates_disabled"
 
+    fish_function.install libexec/"asdf.fish"
```

Comment:
```suggestion
    fish_function.install_symlink libexec/"asdf.fish"
```

to not break existing installs.